tests/integration/core/test_tasking.py

```diff
@@ -54,3 +63,38 @@ def test_unique_enqueue_new_job(default_queue, eda_caplog):
     job = unique_enqueue(default_queue.name, "fake_task", fake_task, number=2)
     assert job.kwargs["number"] == 2
     assert "Enqueing unique job" in eda_caplog.text
+
+
+def test_worker_dab_client(default_queue: Queue):
+    """Test that workers end up with a DABRedis client connection."""
+
+    # Verify if given no connection the worker gets a DABRedis.
+    worker = DefaultWorker([default_queue])
+    assert type(worker.connection) is DABRedis
```

Comment:
Same question about usage of `type` instead `isinstance`

Reply:
@Alex-Izquierdo No.  It would be my first choice but then I saw there is some EDA code that uses type().